hack/lib/util.sh

```diff
@@ -702,6 +702,15 @@ function kube::util::require-jq {
   fi
 }
 
+# outputs md5 hash of $1, works on macOS and Linux
+function kube::util::md5() {
```

Comment:
this is a 1:1 copy of `kube::release:md5`, it's such a trivial method that it didn't seem particularly worth deduping even, but we don't want to source the whole release machinery in update-vendor-licenses.sh

Reply:
(see `build/lib/release.sh`)

I can't find any consumers of `kube::release::md5` but I'm wary of potentially breaking some script elsewhere (k/release??) and removing it would be a somewhat orthogonal change.